examples/cerebro/clearnet/clearnode_client.go

```diff
@@ -282,20 +281,24 @@ type TransactionResponse struct {
 	CreatedAt      time.Time       `json:"created_at"`
 }
 
-func (c *ClearnodeClient) Transfer(destinationTag, assetSymbol string, amount decimal.Decimal) (*TransactionResponse, error) {
+func (c *ClearnodeClient) Transfer(transferByTag bool, destinationValue string, assetSymbol string, amount decimal.Decimal) (*TransactionResponse, error) {
 	if c.signer == nil {
 		return nil, fmt.Errorf("client not authenticated")
 	}
 
 	params := TransferReq{
-		DestinationUserTag: destinationTag,
 		Allocations: []TransferAllocation{
 			{
 				AssetSymbol: assetSymbol,
 				Amount:      amount,
 			},
 		},
 	}
+	if transferByTag {
+		params.DestinationUserTag = destinationValue
+	} else {
+		params.Destination = destinationValue
+	}
```

Comment:
This looks inconvenient to use... What do you think of such (that is takes both `destinationUserTag` and `destination` parameters) `transfer` method API? Maybe it is better to change the underlying `TransferParams` struct and `HandleTransfer` logic respectively?

Reply:
Correct

Reply:
Same as [here](https://github.com/erc7824/nitrolite/pull/288#discussion_r2222190192)